### PR TITLE
Change post register step to keep on serving PatronRegistrationBlock

### PR DESCRIPTION
**Pull request by spaceo**
#### Link to issue

https://reload.atlassian.net/browse/DDFLSBP-615

#### Description

Change post register step to keep on serving PatronRegistrationBlock
Since post register now is handled by the patron registration react app, we can remove the functionality from the cms.

#### Additional comments or questions

Belongs to a trinity together with:
+ https://github.com/danskernesdigitalebibliotek/dpl-design-system/pull/619
+ https://github.com/danskernesdigitalebibliotek/dpl-react/pull/1166

I have a TODO about changing the dependencies of design and react when this has been accepted.

### CODE REVIEW

File: web/modules/custom/dpl_login/src/Controller/DplLoginController.php [hunk 1]
```diff
@@ -81,6 +84,12 @@ public function logout(): TrustedRedirectResponse|RedirectResponse {
       ->toString(TRUE)
       ->getGeneratedUrl();
 
+    if ($current_path = (string) $request->query->get('current-path')) {
+      $redirect_uri = Url::fromUri(sprintf('internal:%s', $current_path), ['absolute' => TRUE])
```
Comment (Dresse) [inline web/modules/custom/dpl_login/src/Controller/DplLoginController.php:88]: Er det `sprintf()` der gjorde forskellen med den rigtige encoding?

File: web/modules/custom/dpl_login/tests/src/Unit/DplLoginControllerTest.php [hunk 2]
```diff
@@ -116,13 +117,15 @@ public function testThatExceptionIsThrownIfLogoutEndpointIsMissing(): void {
...
+    $controller->logout($this->prophesize(Request::class)->reveal());
   }
 
   /**
-   * The user is redirected to external login if everything is ok.
+   * The user is redirected to external login when logging out.
    */
-  public function testThatExternalRedirectIsActivatedIfEverythingIsOk(): void {
+  public function testThatExternalRedirectIsActivatedWhenLoggingOut(): void {
+    $this->markTestSkipped('After logout is handling current-path, this test has to be updated.');
```
Comment (Dresse) [inline web/modules/custom/dpl_login/tests/src/Unit/DplLoginControllerTest.php:127]: Ved ikke om man også skulle tilføje en `todo` her?

**Review by Dresse — APPROVED** [review]
LGTM - Have small suggestion and also a question just because I'm curios 😄

File: web/modules/custom/dpl_login/tests/src/Unit/DplLoginControllerTest.php (hunk 2 above)
Comment (spaceo) [reply web/modules/custom/dpl_login/tests/src/Unit/DplLoginControllerTest.php:127]: 👍

File: web/modules/custom/dpl_login/src/Controller/DplLoginController.php (hunk 1 above)
Comment (spaceo) [reply web/modules/custom/dpl_login/src/Controller/DplLoginController.php:88]: Nope.
Det var denne:
https://github.com/danskernesdigitalebibliotek/dpl-react/pull/1166/commits/d84b2fac4191a0052768bdb567e4569a8a1a85ae

**Comment by spaceo** [comment]
Moved to separate release:
https://github.com/danskernesdigitalebibliotek/dpl-cms/pull/1111

**Pull request closed without merging**